text/1003-deprecation-import-ember-from-ember.md

```diff
@@ -0,0 +1,369 @@
+
+By removing this set of exports, we have an opportunity to shrink some apps (as some APIs are not used), improving the load performance of ember apps -- and we removing all of these gives us a chance to have a better grasp of what we can get rid of permananently.
+
+Many of these APIs already have alternatives, and those will be called out explicitly in the _Transition Path_ below.
+
+## Transition Path
+
+This list is semi-exhaustive, in that it covers _every_ export from 'ember', but may not exhaustivily provide alternatives.
+
+Throughout the rest of this RFC, the following key will be used:
+- 🌐 to mean "this is public API"
+- 🔒 to mean "this is private API"
+- 🧷 to mean "this is protected API"
+- 🫣 to mean "no declared access"
+
+### Testing utilities
+
+APIs for wiring up a test framework (e.g. QUnit, _etc_)
+
+|   | API | Usage: EmberObserver | -- | 
+| - | --- | ----- | -- |
+|🌐 | `Ember.Test` | [A good few](https://emberobserver.com/code-search?codeQuery=Ember.Test&sort=updated&sortAscending=false) | |
+|🌐 | `Ember.Test.Adapter` | [`ember-cli-fastboot` and `@ember/test-helpers`](https://emberobserver.com/code-search?codeQuery=Ember.Test.Adapter&sort=updated&sortAscending=false) | currently available at [`@ember/test`](https://api.emberjs.com/ember/5.6/modules/@ember%2Ftest) |
+|🌐 | `Ember.Test.QUnitAdapter` | [`ember-cli-fastboot`](https://emberobserver.com/code-search?codeQuery=Ember.Test.QUnitAdapter&sort=updated&sortAscending=false) | |
+|🌐 | `Ember.setupForTesting` | [`ember-cli-fastboot`](https://emberobserver.com/code-search?codeQuery=Ember.setupForTesting&sort=updated&sortAscending=false) | | 
+
+These will need to be moved to a module such as `@ember/test`.
+
+### A way to communicate with the ember-inspector
+
+The inspector will be hit especially hard by the removal of these APIs.
+
+A good few already have available imports though.
+
+|   | API | import |
+| - | --- | ------ |
+|🔒| `Ember.meta` | `import { meta } from '@ember/-internals/meta';` |
+|🌐| `Ember.VERSION` | none, we should add one, `@ember/version` |
+|🔒| `Ember._captureRenderTree` | `import { captureRenderTree } from '@ember/debug';` |
+|🔒| `Ember.instrument` | `import { instrument } from '@ember/instrumentation';` |
+|🔒| `Ember.subscribe` | `import { subscribe } from '@ember/instrumentation';` |
+|🔒| `Ember.Instrumentation.*` | `import { * } from '@ember/instrumentation';`  |
+|🫣| `Ember.ViewUtils` | `import * as viewUtils from '@ember/-internals/views';`[^view-utils]  |
+|🔒| `Ember.ViewUtils.getChildViews` | `import { getChildViews } from '@ember/-internals/views';` |
+|🫣| `Ember.ViewUtils.getElementView` | `import { getElementView } from '@ember/-internals/views';` |
+|🔒| `Ember.ViewUtils.getRootViews` | `import { getRootViews } from '@ember/-internals/views';` |
+|🔒| `Ember.ViewUtils.getViewBounds` | `import { getViewBounds } from '@ember/-internals/views';` |
```

Comment:
FWIW, guilty of using this [in ember-bootstrap](https://github.com/ember-bootstrap/ember-bootstrap/blob/master/ember-bootstrap/addon/components/bs-contextual-help.js#L663), for a narrow edge case. Could change that to a proper import, but without that try/catch this could become even riskier, when getting removed eventually!? 🤔